### PR TITLE
docs: Add usbip guide

### PR DESCRIPTION
**Pull request by robin-nitrokey**

### CODE REVIEW

File: docs/usbip.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- Git
+- Rust
+  - See the [`rust-toolchain.toml`][] file in the repository root for the required version.
+- usbip
+  - While USB/IP is also available for other operating systems, we have only tested and used it on Linux.
+
+[`rust-toolchain.toml`]: ../rust-toolchain.toml
+
+## Running the Simulation
+
```
Comment (szszszsz) [inline docs/usbip.md:25]: Recommendation: add a note to check the limitations chapter before running

File: docs/usbip.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Now the device should show up in `lsusb` and you can use e. g. it with `nitropy`.
+
+## Configuration
+
+### Logging
+
+Use the `RUST_LOG` environment variable to activate log messages.
+It uses the `env_logger` syntax, see the [`env_logger` documentation][].
+For example, you could use `RUST_LOG=info,fido_authenticator=trace` to enable all error, warn and info messages as well as the debug and trade messages from `fido-authenticator`.
```
Comment (szszszsz) [inline docs/usbip.md:62]:
```suggestion
For example, you could use `RUST_LOG=info,fido_authenticator=trace` to enable all error, warn and info messages as well as the debug and trace messages from `fido-authenticator`.
```

File: docs/usbip.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+- GNU Make
+- Git
+- Rust
+  - See the [`rust-toolchain.toml`][] file in the repository root for the required version.
+- usbip
+  - While USB/IP is also available for other operating systems, we have only tested and used it on Linux.
+
+[`rust-toolchain.toml`]: ../rust-toolchain.toml
+
```
Comment (szszszsz) [inline docs/usbip.md:23]: Recommendation: you can add here the kernel module we are loading during attach operation (or does it come with the usbip tools package?)

**Review by szszszsz — APPROVED** [review]

File: docs/usbip.md (hunk 3 above)
Comment (robin-nitrokey) [reply docs/usbip.md:23]: In Debian, the module is part of the default kernel installation.  Does it have to be installed separately in other distros?

Comment (szszszsz) [reply docs/usbip.md:23]: I thought so, but I can't find it packaged in Fedora - it must be build by default here too.
Usbip package comes just with the tool, and nothing kernel related.
In that case there is no need to mention it here, my bad.

**Pull request merged**